.travis.yml

```diff
@@ -224,10 +224,13 @@ script:
   # append CXX_STANDARD to CMAKE_OPTIONS if required
   - CMAKE_OPTIONS+=${CXX_STANDARD:+ -DCMAKE_CXX_STANDARD=$CXX_STANDARD -DCMAKE_CXX_STANDARD_REQUIRED=ON}
 
+  # build configuration
+  - CMAKE_OPTIONS+=" -DCMAKE_BUILD_TYPE=Release"
```

Comment:
But doesn't this mean we compile all tests in Release mode?

Reply:
The test configuration with `BUILD_TYPE=Debug` was previously used for coverage check only. This configuration is defined now in GitHub workflows. Others don't set build type explicitly and use the default which is `Release`. I'm not sure wheter the default should be `Release` or `Debug`. Is there a reason to prefer `Debug` here?

Reply:
I see. I never thought about the "default" build type.